src/views/AuthoritiesSearch/AuthoritiesSearch.js

```diff
@@ -279,6 +279,14 @@ const AuthoritiesSearch = ({ children }) => {
     value: index.value,
   }));
 
+  const handleKeyDown = (e) => {
```

Comment:
Actually you already have same function `onSubmitSearch`, so you could pass it into `<SearchTextareaField>` component.

Why it should be done? Right now your test for this action doesn't work as exactly. Because it works for each button which you pressed.

In this case, better to create prop `onSubmitSearch`, for `<SearchTextareaField>`  and make `handleKeyDown` inside this ocmponent

```
cosnt { onSubmitSearch } = props;

const handleKeyDown = (e) => {
  if (e.keyCode === 13) { // better use keyCode instead of key
    onSubmitSearch();
  }
}
```

After that you will see that your test doesn't work, you should change key which you handle